<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html
  PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:exslt="http://exslt.org/common">
   <head>
      <title>Domain: Clinical Genomics - Genotype Topic</title>
      <link rel="stylesheet" type="text/css" href="../../support/css/v3m.css" title="v3m" /><script src="../../support/js/tree.js" type="text/javascript"></script><script src="../../support/js/setTopFrameText.js" type="text/javascript"></script><script src="../../support/js/swapcss.js" type="text/javascript"></script><script type="text/javascript">
				if (find('heading')) setHeadingTextFromFileName(document.location.href,"Genotype Topic");
			</script></head>
   <body>
      <div class="section">
         <div class="header"><span class="icon"><img src="../../support/graphics/DSTU.gif" alt="DSTU1" /></span><span class="number">2</span><span class="title"><a name="POCG_DO000000UV-Genotype-ic">Genotype Topic</a></span><table cellspacing="0" cellpadding="3" border="1">
               <tr>
                  <td align="center"><img src="../../support/graphics/dstulogo.gif" alt="DSTU" /><br />HL7 V3 CG_GENO,R1<br />HL7 Version 3 Standard: Clinical Genomics; Genotype, Release 1<br />January 2007<img width="5" alt="spacer" src="../../support/graphics/spacer.gif" /></td>
               </tr>
            </table>
         </div>
         <div class="body">
            <table>
               <tbody>
                  <tr>
                     <td><a class="custombutton" href="uvcg.htm#contents">Return to Domain Table of Contents</a></td>
                  </tr>
                  <tr>
                     <td colspan="2">
                        <p class="lastEdit">Content Last Edited: 2008-06-04T16:32:56</p>
                     </td>
                  </tr>
               </tbody>
            </table><br /><div class="toc"><span>2.1  <a href="#sbs-POCG_DO000000UV-Genotype-ic">Storyboards</a><br /></span><span>2.2  <a href="#rmims-POCG_DO000000UV-Genotype-ic">Refined Message Information Models</a><br /></span><span>2.3  <a href="#hmds-POCG_DO000000UV-Genotype-ic">Hierarchical Message Descriptions</a><br /></span></div>
            <div class="subSection">
               <div class="header"><span class="icon"> </span><span class="number">2.1</span><span class="title"><a name="sbs-POCG_DO000000UV-Genotype-ic">Storyboards</a></span><hr class="sectionrule" />
               </div>
               <div class="body">
                  <p><a href="#">Go To Top</a></p>
                  <table class="artifactList">
                     <tr>
                        <td class="artifactList_desc">
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e282" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Storyboards (Sorted by Title)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Cystic Fibrosis Genetic Testing(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000020UV01-str" class="">POCG_ST000020UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Pharmacogenomics-Based Clinical Trial &amp; Submission(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000030UV01-str" class="">POCG_ST000030UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Tissue Typing Genetic Testing(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000010UV01-str" class="">POCG_ST000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e2822" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Storyboards (Sorted by Structured Sort Name)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Genotype Cystic Fibrosis(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000020UV01-str" class="">POCG_ST000020UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Genotype Pharmacogenomics(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000030UV01-str" class="">POCG_ST000030UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Genotype Tissue Typing(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000010UV01-str" class="">POCG_ST000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e2823" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Storyboards (Sorted by Display Order)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Cystic Fibrosis Genetic Testing(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000020UV01-str" class="">POCG_ST000020UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Pharmacogenomics-Based Clinical Trial &amp; Submission(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000030UV01-str" class="">POCG_ST000030UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Tissue Typing Genetic Testing(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_ST000010UV01-str" class="">POCG_ST000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                        </td>
                     </tr>
                  </table>
                  <div class="heading"><br /><b class="heading">Reference</b></div>
                  <p>For details on the interpretation of this section, see <a href="../../help/v3guide/v3guide.htm#v3gstory">the storyboard discussion</a> in the Version 3 Guide.
                  </p>
                  <div class="subSubSection">
                     <div class="header"><span class="icon"><img src="../../support/graphics/DSTU.gif" alt="DSTU1" /></span><span class="number">2.1.1</span><span class="title"><a name="POCG_ST000020UV01-str">Cystic Fibrosis Genetic Testing  (POCG_ST000020UV01) </a></span></div>
                     <div class="body">
                        <div class="heading"><br /><b class="heading">Purpose</b></div>
                        <p>The purpose of this storyboard is to illustrate the request and report messaging as well as documents related to Cystic Fibrosis
                           testing.
                        </p>
                        <div class="heading">
                           <p>Diagram</p>
                        </div><a href="graphics/L-POCG_ST000020UV01.gif" target="_blank" style="border:0px">Link to wide graphic (opens in a new window)</a><p></p>
                        <div class="subSubSubSection changed">
                           <div class="header"><span class="icon"> </span><span class="number">2.1.1.1</span><span class="title"><a name="POCG_ST000020UV01-POCG_SN000020UV01-stex">Cystic Fibrosis (POCG_SN000020UV01) </a></span></div>
                           <div class="body">
                              <p><strong>Summary:</strong></p>
                              <ol>
                                 <li>
                                    <p>Patient meets criteria for Cystic Fibrosis (CF) testing.</p>
                                 </li>
                                 <li>
                                    <p>Informed consent signed.</p>
                                 </li>
                                 <li>
                                    <p>Tests ordered.</p>
                                 </li>
                                 <li>
                                    <p>Results received.</p>
                                 </li>
                                 <li>
                                    <p>Results explained.</p>
                                 </li>
                                 <li>
                                    <p>Plan drafted.</p>
                                 </li>
                              </ol>
                              <p><strong>Detail:</strong></p>
                              <p>The patient's ethnicity, medical history, and/or family history (as documented in chart ) trigger the consideration of  genetic
                                 testing for CF mutations.  The CF test is explained to the patient (documented), the patient's questions are  answered (documented),
                                 and the informed consent is signed.  The doctor uses an electronic order entry system (same as  for other lab tests) to order
                                 CF testing (documented).  The lab system receives the order and forwards it to the  Molecular Genetics System (MGS).  A blood
                                 sample is drawn and sent to the Molecular Genetics Laboratory.  The test is  performed and results are provided by a lab technologist
                                 on a "worksheet".  The Molecular Laboratory's Genetic Counselor  reviews the results and the pertinent history and then creates
                                 a report in MGS including the reason for referral,  testing method, results, and interpretation (documented).  The results
                                 and report are reviewed by the Molecular  Laboratory Consultant and released. The report is sent from MGS to the lab system
                                 and also to the EMR system for viewing  by the doctor. The physician and patient discuss the test results (documented), and
                                 appropriate plans are outlined ( documented). 
                              </p>
                              <p><strong>Presentation:</strong></p>
                              <p>Pat Patient is being seen at Care Clinic by Dr. Doorman.  Dr. Doorman reviews Pat's EHR, ethnicity, medical history, and 
                                 family history and determines that Pat should be screened for Cystic Fibrosis mutations.  Dr. Doorman answers Pat's  questions
                                 and Pat signs an informed consent form. 
                              </p>
                              <p>Dr. Doorman orders a Cystic Fibrosis test through Orders Plus (Electronic Order Entry System).  Lab Plus receives the  order
                                 and forwards it to Molecular Genetics Plus (Molecular Genetics System). 
                              </p>
                              <p>Anita Nurse takes a blood sample from Pat and sends it to Molecular Lab Plus (Molecular Genetics Lab). </p>
                              <p>Molly Molecular (Lab Technologist) receives the blood specimen and the test order and performs the Cystic Fibrosis test. 
                                 She records the results on a worksheet. 
                              </p>
                              <p>Carla Counselor (Molecular Genetics Counselor) reviews the results and Pat's history and creates a report in Molecular  Genetics
                                 Plus. 
                              </p>
                              <p>Cory Consultant reviews Carla's work, makes any necessary modifications, and releases the report to Molecular Genetics  Plus
                                 and Lab Plus. 
                              </p>
                              <p>Dr. Doorman looks at Pat's results in the EHR.  Dr. Doorman and Pat meet to discuss the results and appropriate next  steps.
                                 
                              </p>
                              <p><strong>Triggers:</strong></p>
                              <ul>
                                 <li>
                                    <p>Family history</p>
                                 </li>
                                 <li>
                                    <p>Patient has a suspected diagnosis</p>
                                 </li>
                                 <li>
                                    <p>Carrier screening (routinely offered to all pregnant couples)</p>
                                 </li>
                                 <li>
                                    <p>Male infertility (CBAVD - Congenital Bilateral Absence of the Vas Deferens)</p>
                                 </li>
                                 <li>
                                    <p>Ethnicity</p>
                                 </li>
                              </ul>
                              <p><strong>Data:</strong></p>
                              <ol>
                                 <li>
                                    <p>Ethnicity, Medical history, and Family history risk factors.  Examples: </p>
                                    <ul>
                                       <li>
                                          <p>Positive sweat chloride test</p>
                                       </li>
                                       <li>
                                          <p>Pulmonary and/or pancreatic symptoms</p>
                                       </li>
                                       <li>
                                          <p>European ancestry</p>
                                       </li>
                                       <li>
                                          <p>Affected family member</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Informed consent document. </p>
                                    <ul>
                                       <li>
                                          <p>Supplementary educational materials</p>
                                       </li>
                                       <li>
                                          <p>Electronic patient signature</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Lab order </p>
                                    <ul>
                                       <li>
                                          <p>Orderable in formulary</p>
                                       </li>
                                       <li>
                                          <p>Test prerequisites</p>
                                       </li>
                                       <li>
                                          <p>Protocol/workflow</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Lab result </p>
                                    <ul>
                                       <li>
                                          <p>33 mutations (greater than 900 documented mutations in the CF gene)</p>
                                       </li>
                                       <li>
                                          <p>mutation name</p>
                                       </li>
                                       <li>
                                          <p>location (specific exon or intron)</p>
                                       </li>
                                       <li>
                                          <p>Interpretation with explanation, considering</p>
                                       </li>
                                       <li>
                                          <p>reason for referral</p>
                                       </li>
                                       <li>
                                          <p>ethnicity</p>
                                       </li>
                                       <li>
                                          <p>family history</p>
                                       </li>
                                    </ul>
                                 </li>
                              </ol>
                              <p>This storyboard is also supported by the following Domain Analysis Model in UML. 	<a href="graphics/POCg_NA000021.gif" target="_blank" style="border:0"><img class="graphic" src="graphics/T-POCg_NA000021.gif" alt="Domain Analysis Model (UML)" /></a></p>
                              <p>Messages in this storyboard will be derived from models that refine other balloted models, most likely the Lab model (the
                                 refinement process is still a work in progress). The following bird's eye view of such a model is meant for illustration purposes
                                 only <strong>and is not part of the normative ballot.</strong> It demonstrates how the Genotype model fits into that model at the genomic data level based on the storyboard requirements:
                              </p>
                              <p><br /><img class="graphic" src="graphics/t-POCg_gR000024.gif" alt="Cystic Fibrosis Testing RMIM" /><br />Cystic Fibrosis Testing RMIM<br /> describes the Cystic Fibrosis storyboard requirements via an HL7 model which utilizes the Genotype model to convey genomic
                                 data at any level of granularity needed. 
                              </p>
                           </div>
                        </div>
                     </div>
                  </div>
                  <div class="subSubSection">
                     <div class="header"><span class="icon"><img src="../../support/graphics/DSTU.gif" alt="DSTU1" /></span><span class="number">2.1.2</span><span class="title"><a name="POCG_ST000030UV01-str">Pharmacogenomics-Based Clinical Trial &amp; Submission  (POCG_ST000030UV01) </a></span></div>
                     <div class="body">
                        <div class="heading"><br /><b class="heading">Purpose</b></div>
                        <p>The purpose of this storyboard is to illustrate the request and report messaging as well as documents related to Pharmacogenomics-based
                           Clinical Trial &amp; submission.
                        </p>
                        <div class="heading">
                           <p>Diagram</p>
                        </div><a href="graphics/L-POCG_ST000030UV01.gif" target="_blank" style="border:0px">Link to wide graphic (opens in a new window)</a><p></p>
                        <div class="subSubSubSection changed">
                           <div class="header"><span class="icon"> </span><span class="number">2.1.2.1</span><span class="title"><a name="POCG_ST000030UV01-POCG_SN000030UV01-stex">Pharmacogenomics (POCG_SN000030UV01) </a></span></div>
                           <div class="body">
                              <p><strong>Scenario Summary</strong></p>
                              <ol>
                                 <li>
                                    <p>Clinical trial designs with pharmacogenomics components.</p>
                                 </li>
                                 <li>
                                    <p>IRB approval.</p>
                                 </li>
                                 <li>
                                    <p>Patient case report form (CRF) and informed consent form design.</p>
                                 </li>
                                 <li>
                                    <p>Patient recruitment and DNA or tissue sample receiving.</p>
                                 </li>
                                 <li>
                                    <p>Gene selection and variation markers discovery.</p>
                                 </li>
                                 <li>
                                    <p>Regulation compliant clinical genotyping.</p>
                                 </li>
                                 <li>
                                    <p>Clinical demographic, clinical phenotype, and lab test results received.</p>
                                 </li>
                                 <li>
                                    <p>Haplotype or other types of marker assignment to the patients.</p>
                                 </li>
                                 <li>
                                    <p>Statistical analysis to establish the association of the genetic markers to the clinical outcomes.</p>
                                 </li>
                                 <li>
                                    <p>Validation study.</p>
                                 </li>
                                 <li>
                                    <p>Reports generation and communicated between pharmaceutical and biotechnology partners, or submission materials  compiled for
                                       regulatory agencies.
                                    </p>
                                 </li>
                              </ol>
                              <p><strong>Workflow And Process</strong></p>
                              <p>Clinical trials with a therapeutic product or device targeting to the global or niche pharmaceutical industry  area requires
                                 the understanding of the genetic structure of the potential patient population and its interaction with  the drug or treatment.
                                 Therefore the design of the trial involves the pharmacogenomics component. For example, the trial  has to consider the patient
                                 population in terms of geo-ethno background recruited into the trial and the other  covariates that stems from the genomics
                                 science. The sample size needs to be determined based on the understanding or  estimate of the gene variation markers in terms
                                 of the allele frequency (common vs. rare), genetic model (recessive vs.  dominant), and strength of the marker prediction
                                 power.
                                 
                              </p>
                              <p>As with any traditional types of trials, the pharmacogenomics based trials also need to obtain the regulatory  approval and
                                 the patient enrollment needs to follow the standard and GCP (good clinical practice) procedures, such as  using IRB approval
                                 of the design, obtaining patient consent forms for the genomics work, etc. The approved clinical  research centers could be
                                 used to collect the patient clinical data during the dosage dependent trials and the other  monitoring and diagnostics lab
                                 testing will be performed at the accredited labs. 
                                 
                              </p>
                              <p>Patient DNA samples extracted from the relevant tissues will be sent to the clinical regulatory compliant  genotyping facilities.
                                 Genotyping work will be applied to these DNA sample using the pharmacogenomics markers selected  for the trials. The genotype
                                 results will be sent to the data management databases of the trial and statistical analysis  will be performed to discover
                                 the associations between the genetics markers and the clinical outcomes and endpoints.
                                 
                              </p>
                              <p>The discovery results will be then compiled into the clinical trial reports that can be used for further drug  development.
                                 It can also be integrated into the submission packages used for the product application and approval to the  regulatory bodies.
                                 
                              </p>
                              <p><strong>Trigger Events</strong></p>
                              <ul>
                                 <li>
                                    <p>DNA Banking from the previous trials advocates the use of genomics technology to profile the compound and  population.</p>
                                 </li>
                                 <li>
                                    <p>Previous trials or market information indicates the divided responses in the target population to the class of  drugs or treatments.</p>
                                 </li>
                                 <li>
                                    <p>Characterization of the drug target(s) reveals prevalence of the genetic variation in the gene or pathways  involved in the
                                       mode of action (MOA) of the drug or disease susceptibility or progression.
                                    </p>
                                 </li>
                                 <li>
                                    <p>Ab initio trial with pharmacogenomics design.</p>
                                 </li>
                              </ul>
                              <p><strong>Data</strong></p>
                              <ol>
                                 <li>
                                    <p>Patient demographic and clinical observations as reported in CRF (case report form):
                                       
                                    </p>
                                    <ul>
                                       <li>
                                          <p>Age.</p>
                                       </li>
                                       <li>
                                          <p>Gender.</p>
                                       </li>
                                       <li>
                                          <p>Ethnicity.</p>
                                       </li>
                                       <li>
                                          <p>Geographic origins.</p>
                                       </li>
                                       <li>
                                          <p>Dosage.</p>
                                       </li>
                                       <li>
                                          <p>Confounding disease or drug usage.</p>
                                       </li>
                                       <li>
                                          <p>Clinic visit observations: BMI, blood pressure, etc. depending on the therapeutic areas.</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Informed consent document.
                                       
                                    </p>
                                    <ul>
                                       <li>
                                          <p>Genetic testing education materials.</p>
                                       </li>
                                       <li>
                                          <p>Electronic patient signature.</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Lab test
                                       
                                    </p>
                                    <ul>
                                       <li>
                                          <p>Test prerequisites.</p>
                                       </li>
                                       <li>
                                          <p>Protocol/workflow</p>
                                       </li>
                                       <li>
                                          <p>Test result.</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Genomics data
                                       
                                    </p>
                                    <ul>
                                       <li>
                                          <p>Selected genes
                                             
                                          </p>
                                          <ul>
                                             <li>
                                                <p>Functional annotation of the genes.</p>
                                             </li>
                                             <li>
                                                <p>Genomic structure</p>
                                             </li>
                                             <li>
                                                <p>Transcript structure (optional)</p>
                                             </li>
                                             <li>
                                                <p>Protein structure (optional)</p>
                                             </li>
                                             <li>
                                                <p>SNPs</p>
                                             </li>
                                             <li>
                                                <p>Haplotypes</p>
                                             </li>
                                          </ul>
                                       </li>
                                       <li>
                                          <p>Genotyping data
                                             
                                          </p>
                                          <ul>
                                             <li>
                                                <p>Minimal SNP selection</p>
                                             </li>
                                             <li>
                                                <p>SNP marker assay design</p>
                                             </li>
                                             <li>
                                                <p>Assay validation and production </p>
                                             </li>
                                             <li>
                                                <p>HapLocus building and haplotype assignments</p>
                                             </li>
                                             <li>
                                                <p>Allele frequencies.</p>
                                             </li>
                                          </ul>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Association Analysis:
                                       
                                    </p>
                                    <ul>
                                       <li>
                                          <p>Hypothesis generation: marker model (dominant vs. recessive), gene model (mono vs. multiple)</p>
                                       </li>
                                       <li>
                                          <p>Statistics generation: marker raw p values, adjusted p values, odds ratios; regression, multiple comparison,  and permutations.</p>
                                       </li>
                                       <li>
                                          <p>Marker quality control</p>
                                       </li>
                                       <li>
                                          <p>Biological story building</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Reports</p>
                                 </li>
                                 <li>
                                    <p>Validation</p>
                                 </li>
                                 <li>
                                    <p>Submission</p>
                                 </li>
                              </ol>
                              <p>The following HL7 model was developed after the above UML activity diagram and utilizes the Genotype model: </p>
                              <p><strong>IMPORTANT NOTE: The bird's eye view of the model is presented for illustration purposes only and is not part of the normative
                                    ballot. Its main purpose is to illustarte how the Genotype model fits into higher level models such as this.</strong></p><a href="graphics/POCg_gR000031.gif" target="_blank" style="border:0"><img class="graphic" src="graphics/T-POCg_gR000031.gif" alt="T-POCG_GR000031.gif" /></a><p><strong>Model Walk-Through</strong></p>
                              <p>The model is focused on genomic data in the clinical trial arena. It is important to note at this point that all parts of
                                 this model that are not related to genomic data could be replaced by models developed by the HL7 RCRIM TC (Regulated Clinical
                                 Research Information Management) that focuses on clinical trials at large. The 'non-genomic' parts of this model were developed
                                 for the sake of better understanding of the genomic-based processes (e.g., gene selection and haplotype discovery), and for
                                 completeness of the model that might make it more convenient to read. Both HL7 groups hold joint meetings and try to harmonize
                                 their models. A dedicated mechanism in HL7 for sharing parts of models across different groups (domains) is the CMET mechanism
                                 (Common Message Element Type). A CMET model is an HL7 model that can be incorporated into another model in a modular way.
                                 The key for using CMETs effectively is clean modularity. In the HL7 CG group we have been trying to develop a Genotype CMET
                                 - an atomic unit that could be plugged into any model that needs to convey genomic data. So, the Pharmacogenomics DIM model
                                 introduced here is in fact a prototyping framework in which we use the HL7 CG Genotype CMET on the one hand and on the other
                                 hand we strive to harmonize it with the HL7 RCRIM models (CDISC-based). 
                              </p>
                              <p>Having said that, we could start the walk-through of the Pharmacogenomics model:  The left side of the model has the main
                                 participants in this model, i.e., the sponsor (Pharma), the CRO and the Patient. There are 3 more participants I the model
                                 (Device, Collaborator and regulator) and they will be described further on. 
                              </p>
                              <ul>
                                 <li>At the upper left corner we can see the patient object and its related information. Note that this model follows the HL7 V3
                                    RIM (Reference Information Model) core classes and associations. For example, we have a Person entity (green box) who plays
                                    the role (yellow box) of a Patient with participation (cyan box) of being the subject of the PahrmacogenomicsClinicalTrial
                                    Act.
                                 </li>
                                 <li>At the middle left we can see the CRO recruiting patients - a process represented by the receiving the consent form for which
                                    the patient has the participation of legalAuthenticator. 
                                 </li>
                                 <li>At the lower left we can see the Pharma entity taking the role of ClinicalResearchSponsor which has the participation of Performer
                                    of the PharmacogenomicsTrialDesign act. 
                                 </li>
                              </ul>
                              <p>On the right side of the model and continuing to the bottom we can find the analysis and reporting process starting from the
                                 Pharmacogenomics trial results and its analysis, on to the CRO, and further to the sponsor. From there we can see the beginning
                                 of the submission process to the regulator with the validation and other necessary actions.
                              </p>
                              <p>At the heart of the model we find the genomic acts, starting from the top where we can see the main act - the PharmacogenomicsClinicalTrial,
                                 associated with the PharmacogenomicsTesting act (which is an attempt to generalize the storyboard UML class called "Genotyping").
                                 The PharmacogenomicsTesting act can have any number of the Genotype module (supposed to be a CMET), in two modes: 
                              </p>
                              <ul>
                                 <li>downward PharmacogenomicsTesting we see the component association with the Genotype module, where the actual genomic result
                                    data reside;
                                 </li>
                                 <li>to the left of PharmacogenomicsTesting we can see an association with the GeneSelection act which is a fundamental part of
                                    a Pharmacogenomics clinical trial. To represent the genes that were selected we also use the Genotype module, but populate
                                    only the gene identifiers, while in the trial-result-Genotype-modules we can populate any part of the Genotype model (e.g.,
                                    sequencing, gene expression, SNPs, haplotypes, etc.). 
                                 </li>
                              </ul>
                              <p>The PharmacogenomicsTesting is performed by a DNALab which is a role played by the Collaborator entity (these objects are
                                 shown to the right of the act). 
                              </p>
                           </div>
                        </div>
                     </div>
                  </div>
                  <div class="subSubSection">
                     <div class="header"><span class="icon"><img src="../../support/graphics/DSTU.gif" alt="DSTU1" /></span><span class="number">2.1.3</span><span class="title"><a name="POCG_ST000010UV01-str">Tissue Typing Genetic Testing  (POCG_ST000010UV01) </a></span></div>
                     <div class="body">
                        <div class="heading"><br /><b class="heading">Purpose</b></div>
                        <p>The purpose of this storyboard is to illustrate the request and report messaging as well as documents related to tissue typing
                           testing.
                        </p>
                        <div class="heading">
                           <p>Diagram</p>
                        </div><a href="graphics/L-POCG_ST000010UV01.gif" target="_blank" style="border:0px">Link to wide graphic (opens in a new window)</a><p></p>
                        <div class="subSubSubSection changed">
                           <div class="header"><span class="icon"> </span><span class="number">2.1.3.1</span><span class="title"><a name="POCG_ST000010UV01-POCG_SN000010UV01-stex">Tissue Typing (POCG_SN000010UV01) </a></span></div>
                           <div class="body">
                              <p><strong>Overview</strong></p>
                              <p>Over the last decade there is a shift in the routine of typing testing laboratories, from serological based HLA typing  to
                                 molecular based DNA techniques. With a greater understanding of the extent and relevance of HLA variability, there  has been
                                 a move towards increasingly higher levels of resolution in the tissue typing laboratory. DNA Typing relates to  a number of
                                 lab methods whose results allow DNA similarities of donor-recipient, familial relationships or any two  persons to be established
                                 for the purpose of transplantation of bone-marrow and organs, or in forensic scenarios.  It is  done by comparing the characteristic
                                 polymorphic patterns in the Human Leucocytes Antigens (HLA). The main facility for  using such methods is a tissue typing
                                 laboratory. 
                              </p>
                              <p>In most HLA typing applications, it is useful to have allele identification at higher levels of resolution. For  example,
                                 donor-recipient typing in transplant procedures, disease association or anthropology studies should all be  performed at the
                                 highest level of resolution that is practical. In cases of finding a donor for a patient who needs to  go through bone-marrow
                                 transplantation, the DNA typing results provide with better matching accuracy and might support  the prediction of GVH (graft-versus-host)
                                 disease after transplantation or GVM (Graft Versus Malignancy) effect in the  new trend called 'mini-transplant'. In forensic
                                 scenarios it is used in paternity disputes or to check evidences from  the crime scene against suspects' DNA. In forensic
                                 scenarios even lower resolution methods were accepted so far, but  given the new methods accuracy it's likely to see increasing
                                 demand for DNA typing data that better eliminate any doubt. 
                              </p>
                              <p>The information generated with direct DNA sequencing of HLA genes is qualitatively different than with other DNA- based typing
                                 methods. Obtaining the complete sequence of a gene of interest at the nucleotide level provides the  opportunity to assign
                                 allele typing at higher resolution. Sequencing-based typing is one of the latest technologies  applied to histocompatibility
                                 testing providing the highest level of definition and resolution in class I and II HLA  typing. DNA sequencing is now used
                                 in routine HLA typing through recent advances in automated DNA sequencers and  simplified sequencing chemistry. These advances
                                 have allowed greater sample throughput, providing a realistic,  practical, and preferred alternative to other DNA methods,
                                 while offering a far higher level of resolution. 
                              </p>
                              <p><strong>Triggers</strong></p>
                              <ul>
                                 <li>
                                    <p>Tissue Typing for Organ and Bone-Marrow Transplantation</p>
                                 </li>
                                 <li>
                                    <p>Identify people who may be at risk for certain autoimmune diseases</p>
                                 </li>
                                 <li>
                                    <p>Forensic triggers: crime suspects; paternity test; anthropology studies; ethnic crimes</p>
                                 </li>
                              </ul>
                              <p><strong>Presentation </strong></p>
                              <p>Tina Transplant is referred to BMT (Bone Marrow Transplant) Center for transplant workup. Blood samples are drawn from Tina
                                 and her first degree relatives.  The samples are forwarded to the Tissue Typing Lab with the order.
                              </p>
                              <p>Tissue Typing Lab performs Class I HLA testing.  Matches at Class I are further typed for Class II.  A matching analysis is
                                 prepared and the results report is returned to the ordering Center.
                              </p>
                              <p>BMT Center evaluates results: </p>
                              <ul>
                                 <li>
                                    <p>Best Outcome:  Match with First-Degree Relative(s) </p>
                                    <ul>
                                       <li>
                                          <p>Matching relative approached for consent and donation. </p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Alternate Outcome:  No Match with First-Degree Relative(s), Distant Relatives Available </p>
                                    <ul>
                                       <li>
                                          <p>BMT Center determines whether typing of more distant relatives should be considered.  If marriages between relatives occurred,
                                             or Tina has common haplotypes, distant relatives may be approached for consent and sample.  BMT Center sends blood sample
                                             from distant relative to Tissue Typing Lab for Class I testing.  
                                          </p>
                                       </li>
                                       <li>
                                          <p>Tissue Typing Lab identifies matches at Class I.  These matches are further typed for Class II.  A matching analysis is prepared
                                             and the results report is returned to the BMT Center for evaluation.   If an acceptable match is identified, relative is approached
                                             for consent and donation. 
                                          </p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Alternate Outcome:  No Match with Relative(s), Unrelated Donor Needed </p>
                                    <ul>
                                       <li>
                                          <p>BMT Center identifies candidate Donor Bank(s) and sends Tina's typing data to Donor Bank. </p>
                                       </li>
                                       <li>
                                          <p>Donor Bank searches for Class I (low-resolution) matches and returns match(es) to BMT Center for evaluation.  </p>
                                       </li>
                                       <li>
                                          <p>BMT Center identifies best class I matches and requests DNA for those matches from Donor Bank.  </p>
                                       </li>
                                       <li>
                                          <p>Donor Bank retrieves DNA samples and forwards to Tissue Typing Lab. </p>
                                       </li>
                                       <li>
                                          <p>Tissue Typing Lab performs Class II typing.  Results are returned to BMT Center for evaluation. </p>
                                       </li>
                                       <li>
                                          <p>If a promising match is found, BMT Center requests confirmatory test. </p>
                                       </li>
                                       <li>
                                          <p>Donor Bank obtains new blood samples from potential donor and forwards to Tissue Typing Lab. </p>
                                       </li>
                                       <li>
                                          <p>Tissue Typing Lab performs high-resolution Class I and Class II typing, and returns results to BMT Center. </p>
                                       </li>
                                       <li>
                                          <p>BMT Center evaluations results and determines if suitable unrelated match has been identified.  If an acceptable match is
                                             found, the unrelated donor is approached by the Donor Bank for consent and donation. 
                                          </p>
                                       </li>
                                    </ul>
                                 </li>
                              </ul>
                              <p>Once donor has been identified and consented, BMT Center initiates pre-transplant preparatory regimen for Tina, and obtains
                                 marrow from donor. 
                              </p>
                              <p><strong>Data (for transplant patients)</strong></p>
                              <ol>
                                 <li>
                                    <p>Complete medical and family history for the patients; emphasis on patients with special problems or treatments  such as: </p>
                                    <ul>
                                       <li>
                                          <p>Patients who need repeating transplant</p>
                                       </li>
                                       <li>
                                          <p>Patients who are candidates for bone-marrow "mini-transplant"</p>
                                       </li>
                                       <li>
                                          <p>Patients with diseases associated with HLA (e.g., multiple sclerosis)</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Informed consent document </p>
                                    <ul>
                                       <li>
                                          <p>Supplementary educational materials</p>
                                       </li>
                                       <li>
                                          <p>Electronic patient signature</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Lab order </p>
                                    <ul>
                                       <li>
                                          <p>Orderable in formulary</p>
                                       </li>
                                       <li>
                                          <p>Test prerequisites</p>
                                       </li>
                                       <li>
                                          <p>Protocol/workflow</p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p>Lab result (for patient, donor, patient's mother and father if available) </p>
                                    <ul>
                                       <li>
                                          <p>Class I allele sequences: HLA-A, HLA-B, HLA-C</p>
                                       </li>
                                       <li>
                                          <p>Class II allele sequences: HLA-DP, HLA-DQ, HLA-DR</p>
                                       </li>
                                       <li>
                                          <p>Interpretation with explanation</p>
                                       </li>
                                       <li>
                                          <p>The extent of donor-recipient matching</p>
                                       </li>
                                       <li>
                                          <p>The appropriateness of the previous result to the intended treatment (e.g., the new "mini-transplant" might  better succeed
                                             without full matching)
                                          </p>
                                       </li>
                                       <li>
                                          <p>Annotation of sequences with interpretation details</p>
                                       </li>
                                    </ul>
                                 </li>
                              </ol>
                              <p><strong>IMPORTANT NOTE: The following model is presented for illustration purposes only and is not part of the normative ballot. Its
                                    main purpose is to illustarte how the Genotype model fits into higher level models such as this.</strong></p>
                              <p>This storyboard is also supported by the following Domain Analysis Model in UML. 	<a href="graphics/POCg_NA000011.gif" target="_blank" style="border:0"><img class="graphic" src="graphics/T-POCg_NA000011.gif" alt="Domain Analysis Model (UML)" /></a></p>
                              <p>Messages in this storyboard will be derived from models that refine other balloted models such as the Lab model (the refinement
                                 process is still a work in progress). The following bird's eye views of such models are meant for illustration purposes only
                                 <strong>and are not part of the normative ballot.</strong> They demonstrate the modularity of HL7 models and how the Genotype model fits them at the genomic data level based on the
                                 storyboard requirements:
                              </p>
                              <p><a href="graphics/POCg_gR000015.gif" target="_blank" style="border:0"><img class="graphic" src="graphics/T-POCg_gR000015.gif" alt="Bone-Marrow transplantation Tissue Typing RMIM" /></a> describes the use of tissue typing in bone-marrow transplantation, while <a href="graphics/POCg_gR000016.gif" target="_blank" style="border:0"><img class="graphic" src="graphics/T-POCg_gR000016.gif" alt="Tissue Typing Observation RMIM" /></a> describes a tissue typing observation and matching, used by any tissue typing-related scenario (organ transplantation, fatherhood,
                                 forensic, etc.) and it is the level at which the Genotype model is being utilized. 
                              </p>
                           </div>
                        </div>
                     </div>
                  </div>
               </div>
            </div>
            <div class="subSection">
               <div class="header"><span class="icon"> </span><span class="number">2.2</span><span class="title"><a name="rmims-POCG_DO000000UV-Genotype-ic">Refined Message Information Models</a></span></div>
               <div class="body">
                  <table class="artifactList">
                     <tr>
                        <td class="artifactList_desc">
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e825" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Refined Message Information Models (Sorted by Title)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">GeneticLoci(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_RM000050UV01-rmi" class="">POCG_RM000050UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">GeneticLocus(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_RM000010UV01-rmi" class="">POCG_RM000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e8252" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Refined Message Information Models (Sorted by Structured Sort Name)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Genotype Event(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_RM000010UV01-rmi" class="">POCG_RM000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Genotype Event(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_RM000050UV01-rmi" class="">POCG_RM000050UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e8253" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Refined Message Information Models (Sorted by Display Order)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">GeneticLocus(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_RM000010UV01-rmi" class="">POCG_RM000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">GeneticLoci(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_RM000050UV01-rmi" class="">POCG_RM000050UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                        </td>
                     </tr>
                  </table>
                  <div class="heading"><br /><b class="heading">Reference</b></div>
                  <p>For details on the interpretation of this section, see <a href="../../help/v3guide/v3guide.htm#v3grmim">the description of RMIMs</a> in the Version 3 Guide.
                  </p>
                  <p></p>
                  <div class="subSubSection">
                     <div class="header"><span class="icon"><img src="../../support/graphics/DSTU.gif" alt="DSTU1" /></span><span class="number">2.2.1</span><span class="title"><a name="POCG_RM000010UV01-rmi">GeneticLocus  (POCG_RM000010UV01) </a></span></div>
                     <div class="body">
                        <div class="heading">
                           <p>Diagram</p>
                        </div><a style="border:0px solid none;" href="editable/POCg_RM000010UV.htm" target="_blank"><img class="graphic" src="editable/images/T-POCg_RM000010UV.png" alt="" /><br />
                           						Click thumbnail above to open larger graphic in a new window
                           					</a><div class="subheading">Description</div>
                        <table class="structType">
                           <tr>
                              <td class="structType_name"><b><em>Parent:  </em></b></td>
                              <td class="structType_value"><em>Clinical Genomics Domain Information Model (<a href="../../domains/uvcg/uvcg.htm#POCG_DM000023UV-rmi" class="">POCG_DM000023UV</a>)
                                    									</em></td>
                           </tr>
                        </table>The GeneticLocus model describes data relating to a genetic locus, which we propose to be the basic unit of genomic information
                        exchange in healthcare. This model is not meant to be a biological model; rather it is aimed at the needs of healthcare with
                        the vision of personalized medicine in mind. Also it could facilitate the needs of clinical research conducted within the
                        healthcare enterprises. It is the result of our effort to look for the commonalities in each genomic-oriented storyboard that
                        we have been working on (i.e., Tissue Typing, Cystic Fibrosis, BRCA and Pharmacogenomics). Thus, this model contains a subset
                        of the overall Clinical Genomics DIM as it focuses on a single locus. It appears as a CMET in both the Clinical Genomics DSTU
                        as well as in the HL7 Common Domains so that every HL7 group that needs to convey high-resolution genomic data could utilize
                        it . The Genetic Locus might be derived and further constrained by its main subject (e.g., human, animal, and viral) or by
                        type of genomic data (e.g., DNA, Expression and Proteomics). The trial use of this draft standard will guide the derivation
                        process in a way that is most useful to its early adopters.
                        <p><strong>Genotype Scope and Modularity</strong></p>
                        <p>The Genotype model evolved from the work on several storyboards that involve genomic data: Tissue Typing, Cystic Fibrosis,
                           BRCA, and Pharmacogenomics. For example, in the BMT -tissue typing storyboard we have identified three modules of information:
                           the first one is about the messages and documents being exchanged in the BMT tissue-typing use case. The second one zooms
                           in and describes in more detail the unique observations used in tissue-typing, i.e., the individual tissue-typing observation
                           and the matching observation which indicates the level of matching between two individual tissue-typing observations (e.g.,
                           patient and donor). The rationale for this modularity is that the tissue-typing observation could be used in other use cases
                           such as fatherhood testing and forensic cases. The third zoom-in is about the individual genotype that describes a pair of
                           alleles from the same chromosomal locus and is the center of this document.     
                        </p>
                        <p>Figure 1 describes the containment of the above-mentioned modules. It is expected that the genotype module will be plugged
                           into other use cases such as genetic diagnostic and clinical trials.    The same design analysis was conducted with the other
                           storyboards with the focus on making the Genotype module a shared model that can be then used in different messages that will
                           be created to serve the unique business logic of each of the genomic-orient storyboards.    
                        </p>
                        <p><br /><img class="graphic" src="graphics/POCg_gR000001.gif" alt="Figure 1: Modularity in using the Genotype models" /><br />Figure 1: Modularity in using the Genotype models<br /></p><strong>Main characteristics:</strong><ul>
                           <li>
                              <p>The entry point to this model is a GeneticLocus observation which could be associated with a pair of alleles on paternal and
                                 maternal homologous chromosomes.
                              </p>
                           </li>
                           <li>Core observations associated directly with each allele are Sequence, Sequence Variation and Expression. These core classes
                              are also the ones which encapsulate raw genomic data.
                           </li>
                           <li>In addition, sequence, sequence variation and expression data could be associated directly with the GeneticLocus observation
                              if data is not available at the allelic level granularity.
                           </li>
                           <li>In addition to the core classes, other classes hold extracted and derived data such certain types of variations, expression
                              main results, references to as set of loci (e.g., a haplotype that this gene belongs to), and proteomics data (e.g., determinant
                              peptides).
                           </li>
                           <li>Both the GeneticLocus and IndividualAllele observations can recurse to represent relationships to genes/alleles from other
                              loci (e.g., in a biological pathway).
                           </li>
                           <li>The Sequence class could recurse as well to allow the representation of the translational path, i.e., DNA, RNA and Protein
                              sequences, derived from each other.
                           </li>
                        </ul>
                        <p>The following figure show a bird's eye view of the GeneticLocus model:</p>
                        <p><br /><img class="graphic" src="graphics/genotypeFocalAreas.gif" alt="Figure 2: A bird's eye view of the GeneticLocus model." /><br />Figure 2: A bird's eye view of the GeneticLocus model.<br /></p>
                        <p><strong>Model Walk-Through:</strong></p>
                        <p>Notes:</p>
                        <p>* The Use of the 'id', 'code' and 'value' Attributes in the Genotype Model:  </p>
                        <p>The use of these attributes in the various Genotype classes depends on the extent to which the data has being personalized
                           and how different are the results from the known genome. It is also different in those classes that encapsulate raw genomic
                           data.     For example, in the IndividualAllele class, in the case that the patient's allele was fully sequenced and found
                           to be slightly different than the one registered in GenBank or other reference databases, there is no external code to place
                           in one of those attributes, rather the IndividualAllele class is associated with the Sequence class where the individual sequence
                           could be placed in the value attribute.     If it is a new allele indeed, temporary identifiers could be placed in the id
                           attribute until it is registered externally. If, however, that is a known allele, then the 'value' attribute can be populated
                           with the appropriate code from GenBank for example. In this case there isn't much point in populating the Sequence class as
                           it can be retrieved from GenBank, but for self-containment purposes in a specific implementation, it could be that the GenBank
                           sequence will be copied and placed in the Sequence class.     Overall, the 'id' attribute should be used to uniquely identifying
                           that specific instance (possibly using the LSID format). The 'code' attribute should identify the kind of data stored in the
                           'value' attribute, and the 'value' attribute should hold actual data, for example, a genotype characteristic (e.g., heterozygous)
                           or an external gene code from GenBank, dbSNP. In the 'encapsulating' classes (e.g., Sequence, Expression, etc.) the 'value'
                           attribute should hold the bioinformatics markup itself. In the latter case, the code should hold an indication of the exact
                           bioinformatics format used to populate the 'value' attribute.  
                        </p>
                        <p>* Vocabulries</p>
                        <p>All vocabularies presented in the model walk-through below should be considered informative part of the this ballot document.</p>
                        <ul>
                           <li>
                              <p><strong>Entry Point and Locus/Gene/Allele Classes:</strong></p>
                              <ul>
                                 <li>
                                    <p><strong>GeneticLocus</strong></p>
                                    <p>Important note: The name 'GeneticLocus' refers to ALL genomic data and aspects of a specific locus along a chromosomal or
                                       mitochondrial DNA.      
                                    </p>
                                    <p>The GeneticLocus class is the entry point of the GeneticLocus model. A genotype commonly stands for an allele pair - from
                                       paternal and maternal homologous chromosomes. However, in this model it could be that (1) only one allele is associated with
                                       the locus (in cases of insufficient data or interest in one allele only); (2) no alleles are associated with the locus in
                                       cases where the locus' alleles have not been determined but there is a need to represent data related to the locus such as
                                       expression data, variations and even sequences; and (3) multiple allele are available (in case of tumor tissues where several
                                       acquired (somatic) variants are encountered).
                                    </p>
                                    <p><strong>GenotypeLocus Attributes:</strong></p><br /><img class="graphic" src="graphics/genotypeLocusAttr.gif" alt="GenotypeLocusAttr.gif" /><br /><br /><p><strong>Table 1: GenotypeLocus.code</strong></p><br /><img class="graphic" src="graphics/genotypeLocus.code.gif" alt="GenotypeLocus.code.gif" /><br /><br /><p><strong>Table 2: GenotypeLocus.value</strong></p><br /><img class="graphic" src="graphics/genotypeLocus.value.gif" alt="GenotypeLocus.value.gif" /><br /><br /><ul>
                                       <li>
                                          <p><strong>refrence</strong></p>
                                          <p>The reference class represents a related gene that is on a different locus, and still has significant interrelation with the
                                             source gene (this is a  recursive association of GeneticLocus).    Note that it is possible to either expand the associations
                                             of the referred allele, or just indicate its id, assuming that it is detailed elsewhere (and accessible using its id).   
                                             The association class called reference has a typeCode attribute currently set to REFR but we are developing a new vocabulary
                                             with codes like FUNCTIONAL, PHYSICAL, SIGNALING, and METABOLIC_PATHWAY that will be described in subsequent ballot cycles.
                                             
                                          </p>
                                       </li>
                                       <li><strong>AssociatedObservation</strong><p>The AssociatedObservation class associated with GeneticLocus  is a place holder for various observations related to a locus,
                                             for example, a Copy Number value that represents the number of copies of this gene or allele. The class has a shadow associated
                                             with the IndividualAllele class (see below). Both classes share the same vocabulary at this point but might be separated in
                                             future versions.   The code attribute holds the type of Observation, e.g., COPY_NUMBER and the value holds the actual result.
                                             Another example would be code=ZYGOSITY and value could then be either HOMOZYGOTE or HETEROZYGOTE. See tables 3&amp;4 for more
                                             possible codes.   
                                          </p>
                                          <p><strong>Table 3: AssociatedObservation.code</strong></p><br /><img class="graphic" src="graphics/LocusAssociatedObservation.code.gif" alt="LocusAssociatedObservation.code.gif" /><br /><br /><p>Table 4 lists the vocabulary from which codes are drawn to populate the value attribute. The abstract codes in this vocabulary
                                             are the codes from table 3 and thus these two vocabularies will be maintained in synch. 
                                          </p>
                                          <p><strong>Table 4: AssociatedObservation.value</strong></p><br /><img class="graphic" src="graphics/LocusAssociatedObservation.value.gif" alt="LocusAssociatedObservation.value.gif" /><br /><br /></li>
                                    </ul>
                                 </li>
                                 <li><strong>IndividualAllele</strong><p>Important note: The term 'Individual Allele' doesn't refer necessarily to a known variant of the gene, rather it refers to
                                       the patient data regarding the locus that might well contain personal variations (e.g., SNPs w/unknown-significance).
                                    </p>
                                    <p>The GenotypeLocus class is associated with 0 to many alleles represented by the IndividualAllele class. The IndividualAllele
                                       class identifies the specific allele instance (using the id attribute) and optionally specifies its external code (if known)
                                       and the method by which it was identified. 
                                    </p>
                                    <p><strong>IndividualAllele Attributes:</strong></p><br /><img class="graphic" src="graphics/IndividualAlleleAttr.gif" alt="IndividualAlleleAttr.gif" /><br /><br /><p><strong>Table 5: IndividualAllele.value:</strong></p><br /><img class="graphic" src="graphics/IndividualAllele.value.gif" alt="IndividualAllele.value.gif" /><br /><br /><p><strong>Table 6: IndividualAllele.methodCode:</strong></p><br /><img class="graphic" src="graphics/IndividualAllele.methodCode.gif" alt="IndividualAllele.methodCode.gif" /><br /><br /><ul>
                                       <li>
                                          <p><strong>AssociatedObservation (SHADOW )</strong></p>
                                          <p>This is a shadow of the AssociatedObservation class. Please refer to the description above for more details about the use
                                             of this class.    
                                          </p>
                                       </li>
                                       <li>
                                          <p><strong>reference</strong></p>
                                          <p>The reference class represents a related allele of a different locus, and still has significant interrelation with the source
                                             allele (this is a  recursive association of IndividualAllele). See the equivalent class in association with Genetic Locus
                                             for more details.         
                                          </p>
                                       </li>
                                    </ul>
                                 </li>
                              </ul>
                           </li>
                           <li>
                              <p><strong>Encapsulating Classes</strong></p>
                              <ul>
                                 <li>
                                    <p><strong>Sequence </strong></p>
                                    <p>The Sequence class is a generalization of all types of bio sequences (i.e., DNA, RNA, Protein) encapsulating the raw sequencing
                                       results of the DNA, and the derived sequences of the resultant RNA and protein molecules.  The Sequence class has a recursive
                                       relation that makes it possible to nest an RNA sequence within a DNA sequence, and a protein sequence within an RNA sequence.
                                       The relationship type is DRIV (derivation) as the nested Sequence classes are meant to be placeholders for sequences that
                                       were computed from the first Sequence class which is the only "encapsulating" class in this path (by 'first' we mean the one
                                       that is associated directly with the IndividualAllele class).   
                                    </p>
                                    <p><strong>Sequence Attributes:</strong></p><br /><img class="graphic" src="graphics/SequenceAttr.gif" alt="SequenceAttr.gif" /><br /><br /><p><strong>Table 7: Sequence.methodCode</strong></p><br /><img class="graphic" src="graphics/Sequence.methodCode.gif" alt="Sequence.methodCode.gif" /><br /><br /><ul>
                                       <li>
                                          <p><strong>AssociatedProperty</strong></p>
                                          <p>The AssociatedProperty class is a placeholder for various properties that relate to the sequence data, which are supposed
                                             to be extracted (bubbled-up) from the raw expression data encapsulated in the Sequence class. This class is basically a code-value
                                             pair allowing the association of multiple properties with the core sequence class which sets the context of these property
                                             observations in terms of identification and time for example. See discussion about the differences between associated observations
                                             versus properties further on. Table 8 lists the vocabulary from which codes are drawn to populate the code attribute while
                                             table 9 lists the vocabulary from which codes are drawn to populate the value attribute. The two vocabularies are synchronized
                                             in the sense that table 8 codes are the abstract codes in table 9 and each of them defines the vocabulary (nested within the
                                             abstract code) used when that abstract code was selected to populate the code attribute.
                                          </p>
                                          <p><strong>Table 8: SequenceProperty.code</strong></p><br /><img class="graphic" src="graphics/SequenceProperty.code.gif" alt="SequenceProperty.code.gif" /><br /><br /><p><strong>Table 9: SequenceProperty.value</strong></p><br /><img class="graphic" src="graphics/SequenceProperty.value.gif" alt="SequenceProperty.value.gif" /><br /><br /></li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p><strong>SequenceVariation</strong></p>
                                    <p>The class SequenceVariation is a generalization of all variation types, i.e., in all molecules (DNA, RNA, Protein) and of
                                       all types within each molecule (e.g., in DNA: SNP, Mutation, etc.). Note: This class replaces the Polymorphism hierarchy in
                                       previous versions.
                                    </p>
                                    <p><strong>SequenceVariation Attributes:</strong></p><br /><img class="graphic" src="graphics/SequenceVariationAttr.gif" alt="SequenceVariationAttr.gif" /><br /><br /><p><strong>Table 10: SequenceVariation.interpretationCode</strong></p><br /><img class="graphic" src="graphics/SequenceVariation.interpretationCode.gif" alt="SequenceVariation.interpretationCode.gif" /><br /><br /><ul>
                                       <li>
                                          <p><strong>AssociateProperty</strong></p>
                                          <p>The class AssociatedProperty is a place holder for various properties that relate to a sequence variation, for example, position,
                                             length, region, reference and more. It replaces the distinct observations we had in previous versions of the Genotype model.
                                             This class is basically a code-value pair allowing the association of multiple properties with the core variation class which
                                             sets the context of these property observations in terms of identification and time for example. See discussion about the
                                             differences between associated observations versus properties further on. Table 11 list the vocabulary from which codes are
                                             drawn to populate the code attribute while table 12 lists the vocabulary from which codes are drawn to populate the value
                                             attribute. The two vocabularies are synchronized in the sense that table 11 codes are the abstract codes in table 12 and each
                                             of them defines the vocabulary (nested within the abstract code) used when that abstract code was selected to populate the
                                             code attribute.
                                          </p>
                                          <p><strong>Table 11: SequenceVariationProperty.code</strong></p><br /><img class="graphic" src="graphics/SequenceVariationProperty.code.gif" alt="SequenceVariationProperty.code.gif" /><br /><br /><p><strong>Table 12: SequenceVariationProperty.value</strong></p><br /><img class="graphic" src="graphics/SequenceVariationProperty.value.gif" alt="SequenceVariationProperty.value.gif" /><br /><br /></li>
                                       <li>
                                          <p><strong>TagSNP</strong></p>
                                          <p>The characteristic of a SNP as a 'Tag SNP' is one of the SequenceVariation properties and thus was removed from this version
                                             of the model and was added to the vocabulary.
                                          </p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p><strong>Expression</strong></p>
                                    <p>The class Expression is a generalization of all types of expression data (RNA and protein). Its code attribute identifies
                                       the type of expression data it carries. This class is one of the Genotype's encapsulating classes, that is, it holds in its
                                       value attribute portions of relevant bioinformatics markup (e.g., MAGE-ML for gene expression data), complying with constrained
                                       schemas of the full-fledged markups. In such a case, the code attribute holds the exact reference to the contained bioinformatics
                                       schema which the value's content should comply with. Note that the association cardinality between this class and its source
                                       class IndividualAllele is zero to many. The idea here is to be able to represent gene expression over multiple experiments
                                       for the same allele under perhaps differing clinical environments and with differing expression. If this association is traversed
                                       several times then it's important to populate the id &amp; effectiveTime attributes so that each object of that class will be
                                       distinguished clearly and identified uniquely. 
                                    </p>
                                    <p><strong>Expression Attributes:</strong></p><br /><img class="graphic" src="graphics/ExpressionAttr.gif" alt="ExpressionAttr.gif" /><br /><br /><ul>
                                       <li>
                                          <p><strong>AssociatedProperty</strong></p>
                                          <p>The AssociatedProperty class is a place holder for various properties that relate to expression data, for example, normalized
                                             intensity, qualitative indication, p-value and more, which are supposed to be extracted (bubbled-up) from the raw expression
                                             data encapsulated in the Expression class. The AssociatedProperty class replaces the distinct observations we had in previous
                                             versions of the model. This class is basically a code-value pair allowing the association of multiple properties with the
                                             core expression class which sets the context of these property observations in terms of identification and time for example.
                                             See discussion about the differences between associated observations versus properties further on. Table 13 list the vocabulary
                                             from which codes are drawn to populate the code attribute while table 14 list the vocabulary from which codes are drawn to
                                             populate the value attribute. The two vocabularies are synchronized in the sense that table 13 codes are the abstract codes
                                             in table 14 and each of them defines the vocabulary (nested within the abstract code) used when that abstract code was selected
                                             to populate the code attribute.   
                                          </p>
                                          <p><strong>Table 13: ExpressionProperty.code</strong></p><br /><img class="graphic" src="graphics/ExpressionProperty.code.gif" alt="ExpressionProperty.code.gif" /><br /><br /><p><strong>Table 14: ExpressionProperty.value</strong></p><br /><img class="graphic" src="graphics/ExpressionProperty.value.gif" alt="ExpressionProperty.value.gif" /><br /><br /><p><strong>Table 15: ExpressionProperty.methodCode</strong></p><br /><img class="graphic" src="graphics/ExpressionProperty.methodCode.gif" alt="ExpressionProperty.methodCode.gif" /><br /><br /></li>
                                    </ul>
                                 </li>
                              </ul>
                           </li>
                           <li>
                              <p>Other Classes and Proteomics</p>
                              <ul>
                                 <li>
                                    <p><strong>GeneticLoci</strong></p>
                                    <p>The Haplotype class was replaced by the GeneticLoci class which is a generalization of a haplotype and other sets of loci.
                                       The class could carry essential data about the loci or just hold an id of Genetic Loci instance complying with the HL7 Clinical
                                       Genomics Genetic Loci model (see further on in this document).
                                    </p>
                                    <p><strong>GeneticLoci Attributes:</strong></p><br /><img class="graphic" src="graphics/HaplotypeAttr.gif" alt="HaplotypeAttr.gif" /><br /><br /><ul>
                                       <li>
                                          <p><strong>Translocation</strong></p>
                                          <p>This class was removed and the term was added to the vocabulary of the AssociatedProperty class associated with the SequenceVariation
                                             class.
                                          </p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p><strong>Polypeptide and DeterminantPeptide</strong></p>
                                    <p>The Sequence class could be associated with its resulted or corresponding polypeptides (represented by the Polypeptide class)
                                       as well as with the determinant peptides  (represented by the DeterminantPeptide class). Note that the Sequence class has
                                       a recursive relation and it is possible to nest an RNA sequence within a DNA sequence, and a protein sequence within an RNA
                                       sequence. The Polypeptide could then be associated with the protein sequences or directly with any of the above levels. Also,
                                       it is possible to associate the DeterminantPeptide with the Polypeptide class or directly with the Sequence class. Both classes
                                       (Polypeptide and DeterminantPeptide) could be associated with several instances of the ClinicalPhenotype classes.     
                                    </p>
                                    <p>The proteomics classes in this model represent protein data that one is deriving from the sequences (by means of computational
                                       biology) and is not measuring directly in the subject. The latter could be represented as regular lab results (using the HL7
                                       Lab specs), which could be referenced in the GeneticLocus instance as if they were phenotype observations.     
                                    </p>
                                    <p>A common case for the use of proteomics in this model could be as follows: Checking whether an amino acid change would result
                                       from the variant; if so - whether the new amino acid change is to an amino acid of a different size or charge state that would
                                       likely change the shape of the active region of the protein; how far the change is from the active site; whether the change
                                       is in a regulator region, and so forth. These observations could then be associated to phenotypic data. 
                                    </p>
                                    <p>For example, consider the following case described in OMIM:  "Despite the dramatic responses to EGFR inhibitors in patients
                                       with non-small cell lung cancer, most patients ultimately have a relapse. Kobayashi et al. (2005) reported a patient with
                                       EGFR-mutant, gefitinib-responsive, advanced non-small cell lung cancer who had a relapse after 2 years of complete remission
                                       during treatment with gefitinib. The DNA sequence of the EGFR gene in his tumor biopsy specimen at relapse revealed the presence
                                       of a second mutation {131550.0006}. Structural modeling and biochemical studies showed that this second mutation led to the
                                       gefitinib resistance." (OMIM *131550)    
                                    </p>
                                    <p><strong>Polypeptide and DeterminantPeptide Attributes:</strong></p><br /><img class="graphic" src="graphics/PolypeptideAttr.gif" alt="PolypeptideAttr.gif" /><br /><br /></li>
                              </ul>
                           </li>
                           <li>
                              <p>To Phenotype and Beyond...</p>
                              <ul>
                                 <li>
                                    <p><strong>ClinicalPhenotype</strong></p>
                                    <p>The 'wrapper' class ClinicalPhenotype is associated with the SequenceVariation class and includes a choice of populating an
                                       ObservedClinicalPhenotype class internally (i.e., within the Genotype instance), externally (ExternalObservedClinicalPhenotype)
                                       or pointing to a KnownClinicalPhenotype. Note that this class is just a 'warpper' (i.e., classCode = "ActContainer") of the
                                       actual phenotypes and thus has no attributes of its own.
                                    </p>
                                 </li>
                                 <li>
                                    <p><strong>ExternalObservedClinicalPhenotype</strong></p>
                                    <p>The difference between ObservedClinicalPhenotype and ExternalObservedClinicalPhenotype is that the former represents a phenotype
                                       that the genomic data creator chose to include within the genotype instance itself, while the latter represents a phenotype
                                       that exists externally to the genotype instance, for example residing in the patient record's problem list. This class has
                                       an id  (mandatory attribute) that serves as a pointer to the target observation.  Note that the typeCode attribute is set
                                       to ActRelationshipExternalReference domain which is an "x_" value set  dedicated to representing types of external references.
                                       For example, it could be set to "XCRPT" which means that the source observation is an excerpt from the target observation.
                                       In our case, this could be used when you might want to refer to a clinical observation in a patient's medical record, but
                                       also would like to show a few of its essential data items in the GeneticLocus instance (in the ClinicalPhenotype object).
                                    </p>
                                 </li>
                                 <li>
                                    <p><strong>KnownClinicalPhenotype</strong></p>
                                    <p>The difference between KnownClinicalPhenotype and ObservedClinicalPhenotype is that while observed phenotype means a phenotype
                                       that was observed in the patient, KnownClinicalPhenotype is in fact a piece od knowledge, i.e., it is known to be associated
                                       with the genomic data but not necessarily in this specifc patient to whom the genomic data belongs. The class KnownClinicalPhenotype
                                       has a definitional mood (i.e., taken from or pointing to a description of such a disease in some form of a master file, ontology,
                                       lookup table, etc., separated from the patient data) to reflect the difference from the event mood of the ObservedClinicalPhenotype
                                       class.
                                    </p>
                                 </li>
                                 <li>
                                    <p><strong>pertinentInformation</strong></p>
                                    <p>This ActRelationship class (named pertinentInformation) represents the association of a genomic observation with a number
                                       of phenotype (clinical) observations. Its mandatory attribute typeCode holds the semantics of what is the type of this association.
                                       It is defined as &lt;=PERT which means that any code in the PERT sub-hierarchy of the HL7 ActRelationshipType Vocabulary is permitted
                                       here. For a full list of these codes, please refer to the Vocabulary chapter in the HL7 V3 Ballot Package. Appendix E lists
                                       codes from all branches of the ActRelationshipType Vocabulary (PERT and others) that seem possible candidates for use in Clinical
                                       Genomics. This work is still under progress and we might propose this list as well as our own codes to be a new x_domain in
                                       ActRelationshipType Vocabulary. 
                                    </p>
                                 </li>
                              </ul>
                           </li>
                        </ul>
                        <p><strong>Miscellaneous Issues</strong></p>
                        <ul>
                           <li>
                              <p><strong>Association types: </strong></p>
                              <p>Association types (ActRelationship typeCode) are consistent with the following principles:</p>
                              <ul>
                                 <li>Encapsulating classes are components (COMP typeCode) of the IndividualAllele and GeneticLocus classes, while... </li>
                                 <li>Bubbled-up classes are derivations (DRIV typeCode) of encapsulating classes; </li>
                                 <li>Clinical phenotypes are pertinent to (PERT typeCode) genomic classes (Note: in previous versions this was fixed to "caused
                                    by" but it was too restricted. The code CAUS is part of the PERT sub-vocabulary and thus could be still used, but other codes
                                    are available as well)
                                 </li>
                                 <li>Non patient-specific data items are defined (INST typeCode) by classes with mood code = DEF (definitional), that is, defined
                                    and described outside of the patient medical records, in some kind of master file, dictionary , ontology, etc.;   
                                 </li>
                                 <li>Alleles in one GeneticLocus refer (REFR typeCode) to alleles in another locus. Nevertheless, we develop a vocabulary for such
                                    types of relationship, which will be proposed for RIM harmonization as a new domain in ActRelationshipType Vocabulary (see
                                    table 16 for a first draft).   
                                 </li>
                              </ul>
                              <p><strong>Table 16: reference.typeCode</strong></p><br /><img class="graphic" src="graphics/reference.typeCode.gif" alt="reference.typeCode.gif" /><br /><br /></li>
                           <li>
                              <p><strong>Observation Interpretation vs. Observation Relationships:</strong></p>
                              <p>There is a subtle distinction between observation interpretation codes and ActRelationship types between genomic and clinical
                                 observations. The observation interpretation is in fact a piece of knowledge, not necessarily shown as phenotypes of that
                                 genotype in the patient. If the interpretation code is DELETERIOUS (see table 8) then the observation could be complemented
                                 by associating it to the KnownAssociatedDisease class which is also a non-patient specific data in DEF mood Code. In contrast,
                                 the phenotype is an actual observation valid for this patient, which is associated with a genomic observation.
                              </p>
                           </li>
                           <li>
                              <p><strong>Bioinformatics formats:</strong></p>
                              <p>In general, we use bioinformatics formats in the Genotype model to feature the encapsulation of raw genomic data such as sequencing,
                                 expression and proteomic data. To enable the embedding of such data accepted from labs that work with bioinformatics formats,
                                 it is possible to assign specific XML portions into the Sequence and Expression value attributes (as well as into SequenceVariation).
                                 This encapsulation of 'foreign' markup is made possible due to the use of the HL7 ED (Encapsulating Data) Data Type which
                                 is defined as follows:  "ED holds data that is primarily intended for human interpretation or for further machine processing
                                 which is outside the scope of HL7.  ED includes unformatted or formatted written language, multimedia data, or structured
                                 information as defined by a different standard (e.g., XML-signatures.)"    
                              </p>
                              <p>The use of the XML bioinformatics markups is restricted, that is, not all tags are allowed, rather only a subset which relates
                                 to a specific patient and include the information pertinent to healthcare. The restrictions on those external XML standards
                                 are specified elsewhere but a draft of a constrained BSML schema for sequencing data is presented in Appendix C.  For more
                                 details about the rationale behind this mixture of HL7 and bioinformatics markup, see the section "Coexistence of HL7 Classes
                                 and Bioinformatics Markup".    
                              </p>
                           </li>
                           <li>
                              <p><strong>Validation:</strong></p>
                              <p>The use of external markup in HL7 messages requires that a receiver of an HL7 instance that contains a Genotype instance,
                                 will carry out a 'double-validation' process: first step is to validate the instance against the HL7 message specification
                                 (of which the Genotype schema is part of) and the second phase is to validate the content of those value attributes against
                                 their respective content models. The valid content models of the Sequence and Expression value attributes will be an integral
                                 part of the entire Genotype specification, but at this point it is still considered informative. 
                              </p>
                           </li>
                           <li>
                              <p><strong>Associated Properties / Observations and the Harmonization Proposals:</strong></p>
                              <p>In previous versions we have coped with the reluctance of both the HL7 RIM Harmonization process and the HL7 Clinical Genomics
                                 group to nail down common attributes of genomic observations by adding new classes and attributes to the RIM, by elaborating
                                 on the SequenceVariation and Expression properties and creating two new Observation classes (SequenceVariationProperty and
                                 ExpressionProperty) to be placeholders for each of theses properties. For example, the proposed 'length' property of a possible
                                 SequenceVariation new RIM class could be represented by an object of the SequenceVariationProperty class which only has code
                                 and value attributes. The code will indicate that this observation describes the position of the variation and the value attribute
                                 holds the position itself. The assumption is that this observation is an integral part of the parent observation with the
                                 same effective time. It could be identified only by going through the source variation object. In contrast, we also had the
                                 LocusAssociatedObservation class which is a place holder for associated observations such as copy number, zygosity, dominancy
                                 and gene family. These observations are independent observations that do have an id, effective time and method code.     
                              </p>
                              <p>In this update #1 of the approved DSTU, the associated properties/observation classes were consolidated to two classes: AssociatedProperty
                                 and AssociatedObservation. Instead of having specific class names (e.g., SequenceVariationProperty), all core classes now
                                 have these two generic classes coming off them. It makes the model simpler (but put the burden on the parsing to understand
                                 the context of each associated property/observation).     The basic difference between associated properties and associated
                                 observations is that an associated property should have been (and eventually may be) part of the parent class attributes.
                                 It's an inherent part of the parent observation and thus doesn't have id, time stamp, method, performer, etc. They 'inherent'
                                 all these attributes from their parent. An associated observation, in contrast, is an independent observation and a component
                                 of its parent class.   
                              </p>
                           </li>
                        </ul>
                        <p><strong>Coexistence of HL7 Objects and Bioinformatics Markup:</strong></p>
                        <p>When exploring the Genotype model one could identify the use of bioinformatics markup such as MAGE for gene expression and
                           BSML for DNA sequencing. Also, a few of the HL7 Classes such as the property classes are overlapping the elements of the bioinformatics
                           markup in a way that it is possible to find a SNP represented in both the Sequence class as well as in the SequenceVariationProperty
                           class. The question then arises, what are the relationships of the two and how do they coexist? The following are a few points
                           to note about that issue:
                        </p>
                        <ul>
                           <li>HL7 mission is to develop message/document specs that will be used in healthcare practice. The mission of the HL7 Clinical
                              Genomics SIG is to develop message/document specs where genomic data are involved (e.g., Tissue Typing, Genetic Testing, and
                              Clinical Trials).
                           </li>
                           <li>Bioinformatics communities develop models/markups and are usually not ANSI-Accredited SDOs and thus cannot sanction and maintain
                              these formats. Naturally, their orientation is more towards research and the needs of information exchange between research
                              facilities, data mining and statistical analysis tools.
                           </li>
                           <li>The HL7 Clinical Genomics SIG attempts to constrain existing bioinformatics markups and embed them in the HL7 model. This
                              effort brings up the issue of overlapping between the HL7 core Reference Information Model (RIM) classes &amp; data types from
                              which we derive the Genotype model and embedded bioinformatics markups. One approach is not allowing overlaps. For example,
                              if we model a SNP in an HL7 class, and also allow BSML extractions to be embedded in the Sequence class, then we should constrain
                              the BSML extractions so that it will not hold SNP data (in the Isoform tag), rather those parts of the BSML (if exist) should
                              be transformed and moved to HL7 classes.
                           </li>
                           <li>Another approach is seeing the overlap as beneficial: The bioinformatics markups are kind of raw data that might not always
                              get into the HL7 Clinical Genomics actual instances, rather might be only referenced from the HL7 instance as supporting evidences.
                              When it does get into an HL7 instance then we have a blend of HL7 classes and embedded genomic markup. This situation is analogous
                              to the raw imaging data. The average referring physician doesn't dig into the dozens of CT images taken in a study, rather
                              look at the already designated ROIs or just read the radiologist report.
                           </li>
                           <li>A similar situation might exist in the Clinical Genomics specification: many clinicians might not be able to grasp the raw
                              genomic data represented in the bioinformatics markup but it could be helpful to have parts of it in the HL7 instances for
                              purposes like evidence availably and machine-processabilty. The HL7 classes should be seen as representing the digest of the
                              raw genomic data which is most pertinent to the healthcare practice itself. There is a space here for applications that might
                              parse the bioinformatics markup, intelligently populate the HL7 classes and then associate them with the appropriate clinical
                              data.
                           </li>
                           <li>The HL7 classes in the Genotype model have the advantage of being better tied with the other HL7 classes in the patient record
                              (e.g., in a problem/allergy list) and thus serves better the ability to link individual genomic data to the clinical data
                              of that individual.  Note that bioinformatics models also include clinical data so this poses an overlapping problem in the
                              other direction.
                           </li>
                           <li>The integration applications that recognize bioinformatics markups could integrate various genomic sources into healthcare
                              standards used for patient care, so there will not be one winning format necessarily. The issue should not be which bioinformatics
                              model is the "best fit with HL7 Reference Information Model", rather, how could we develop mechanisms to digest data in various
                              representations and link them to the HL7 RIM for the benefit of personalized medicine.
                           </li>
                           <li>For example, the BSML markup is fairly simple and the HL7 SIG has done some work on BSML in constraining it to be embedded
                              in HL7 classes. XML samples are available to show how the BSML is embedded in an HL7 message of the tissue typing use case.
                              The constraining process of BSML made sure that data items related to presentation elements are not included and that the
                              instance includes one and only one patient's data and this patient is uniquely identified. In contrast, PharmaGKB has a repository
                              orientation like GenBank with actual data available for retrieval. This could be a data source of generic knowledge that can
                              be accessed by healthcare information systems. In HL7 these repositories are usually treated as controlled coding schemes.
                              They need to be recognized by the HL7 Vocabulary Committee and then any HL7 instance can reference terms, identifiers and
                              data from that taxonomy.
                           </li>
                        </ul>
                        <p>Figure 3 shows a conceptual workflow where the above co-existence takes place and executed step-wise. Figure 4 shows an example
                           taken form the sequencing type of data: the most clinically-significant SNPs are bubbled-up from the raw sequencing data and
                           being associated with clinical phenotypes. For illustration of the latter scenario, see sample on the EGRF (described in the
                           samples appendix B), courtesy of the NHANES project carried out in the USA CDC.<br /><img class="graphic" src="graphics/POCg_gR000002.gif" alt="Figure 3: Bubbling up the most clinically-significant portions of the raw genomic data embedded in the encapsulating objects." /><br />Figure 3: Bubbling up the most clinically-significant portions of the raw genomic data embedded in the encapsulating objects.<br /></p>
                        <p><br /><img class="graphic" src="graphics/POCg_gR000003.gif" alt="Figure 4: Bubbling up the most clinically-significant SNPs of the raw sequencing data embedded in the encapsulating objects." /><br />Figure 4: Bubbling up the most clinically-significant SNPs of the raw sequencing data embedded in the encapsulating objects.<br /></p>
                        <p></p>
                        <table class="artifactList">
                           <caption>Contained Hierarchical Message Descriptions</caption>
                           <tr class="">
                              <td class="artifactList_desc">Genotype Event</td>
                              <td class="artifactList_code"><a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_HD000010UV01-hmd" class="">POCG_HD000010UV01</a></td>
                           </tr>
                        </table>
                        <p></p>
                     </div>
                  </div>
                  <div class="subSubSection">
                     <div class="header"><span class="icon"><img src="../../support/graphics/DSTU.gif" alt="DSTU1" /></span><span class="number">2.2.2</span><span class="title"><a name="POCG_RM000050UV01-rmi">GeneticLoci  (POCG_RM000050UV01) </a></span></div>
                     <div class="body">
                        <div class="heading">
                           <p>Diagram</p>
                        </div><a style="border:0px solid none;" href="editable/POCg_RM000050UV.htm" target="_blank"><img class="graphic" src="editable/images/T-POCg_RM000050UV.png" alt="" /><br />
                           						Click thumbnail above to open larger graphic in a new window
                           					</a><div class="subheading">Description</div>
                        <table class="structType">
                           <tr>
                              <td class="structType_name"><b><em>Parent:  </em></b></td>
                              <td class="structType_value"><em>Clinical Genomics Domain Information Model (<a href="../../domains/uvcg/uvcg.htm#POCG_DM000023UV-rmi" class="">POCG_DM000023UV</a>)
                                    									</em></td>
                           </tr>
                        </table>The GeneticLoci model allows the representation of data relating to a set of loci along the DNA material. The set of loci
                        could be of differnet types, for example, a haplotype (allele or SNP), a genetic profile, a biological pathway, a set of genetic
                        test results which contains results of multiple genes, etc. The Genetic Loci model utlizes the Genetic Locus model as a CMET.
                        <p><strong>Model Walk-Through:</strong></p>
                        <ul>
                           <li>
                              <p><strong>Entry Point and Locus/Gene/Allele Classes:</strong></p>
                              <ul>
                                 <li>
                                    <p><strong>GeneticLoci</strong></p>
                                    <p>The entry point is a GeneticLoci class allowing the representation of the type of this loci group (e.g., allele haplotype
                                       like in tissue typing of HLA antigens) and an optional code for identifying the loci set (if available).
                                    </p>
                                    <p><strong>GeneticLoci Attributes:</strong></p><br /><img class="graphic" src="graphics/geneticLociAttr.gif" alt="GeneticLociAttr.gif" /><br /><br /><ul>
                                       <li>
                                          <p><strong>LociChoice</strong></p>
                                          <p>A GeneticLoci instance consists of zero to many GeneticLocus CMETs or other GeneticLoci classes. This recursive structure
                                             allows the representation of a complex set of genetic loci as comprised of 'sub' genetic loci sets. The actual genetic/genomic
                                             data is represented through GeneticLocus instances at any level of nesting required.   
                                          </p>
                                       </li>
                                       <li><strong>AssociatedObservation</strong><p>The AssociatedObservation class associated with GeneticLoci  is a place holder for various observations related to the set
                                             of loci that have been observed independently of the parent observation. This is a generic catcher for data that can be placed
                                             in any other class in this model. Population of the code &amp; value attributes of this class is controlled by a vocabulary which
                                             is still under development.   
                                          </p>
                                       </li>
                                       <li><strong>Interpretation</strong><p>In cases where there is an interpretation to the entire set of loci data, it is possible to use the interpretationCode attribute
                                             of the GeneticLoci class or populate the associated phenotypes (the phenotype model is identical to that in the GeneticLocus
                                             model and might be packaged as an internal CMET for better modularity).
                                          </p>
                                       </li>
                                       <li><strong>Raw genomic data and the value attribute:</strong><p>While the value attribute typically holds a common code of this set of loci, it can also hold raw genomic data for the entire
                                             set of loci. For example, if the set of loci is a gene expression assay, the value attribute can hold the part of the MAGE-ML
                                             xml that holds data on all loci. 
                                          </p>
                                       </li>
                                    </ul>
                                 </li>
                              </ul>
                           </li>
                           <li>
                              <p><strong>Other Classes</strong></p>
                              <ul>
                                 <li>
                                    <p><strong>GeneticReportDocument </strong></p>
                                    <p>A report document that summarizes the data of a genetic loci set could be associated with the GeneticLoci class and is of
                                       type DOCCLIN which represents in HL7 a CDA type (Clinical Document Architecture). It is possible to actually embed an entire
                                       CDA document in the text attribute of the GeneticReportDocument class or just point to this document through the id attribute.
                                       Other related reports (e.g., follow-up's, addenda) could be associated as well. For more details about the use of clinical
                                       documents, please refer to the CDA and the Medical Records specification in the HL7 V3 Ballot Package.   
                                    </p>
                                    <ul>
                                       <li>
                                          <p><strong>sequelTo</strong></p>
                                          <p>This is a recursive association to the GeneticReportDocument class, which allows the representation of several documents that
                                             relate to one another and to the Genetic loci of course. For example, an addendum or follow-up document to the first summary.
                                             
                                          </p>
                                       </li>
                                    </ul>
                                 </li>
                                 <li>
                                    <p><strong>Participants</strong></p>
                                    <p>In general, a group of optional participants are associated with both the GeneticLoci and the GeneticTestOrder classes allowing
                                       the recording of participants of the order act as well as the results fulfilling that order:
                                    </p>
                                    <ul>
                                       <li>
                                          <p><strong>recordTarget</strong></p>
                                          <p>The record target indicates whose medical record holds the documentation of this act (i.e., the order or the results). This
                                             is especially important when the subject of a service is not the patient himself. Note that the subject can be overridden
                                             in certain GeneticLocus instances. This could be useful when describing for example genomic data relating to various specimens
                                             (healthy and tumor tissues) or relating to virus, as part of a genetic testing of a patient who carries this virus.
                                          </p>
                                       </li>
                                       <li>
                                          <p><strong>author</strong></p>
                                          <p>Author of the data.</p>
                                       </li>
                                       <li>
                                          <p><strong>performer</strong></p>
                                          <p>Performer of the observations.</p>
                                       </li>
                                       <li>
                                          <p><strong>verifier</strong></p>
                                          <p>Verifier of the observations.</p>
                                       </li>
                                       <li>
                                          <p><strong>informationRecipient</strong></p>
                                          <p>To whom the data should be sent.</p>
                                       </li>
                                    </ul>
                                 </li>
                              </ul>
                           </li>
                        </ul>
                        <p><strong>The GeneticTestOrder model (for comments only)</strong></p>
                        <p><strong>General:</strong></p>
                        <p>The GeneticLoci class might be representing a fulfillment of a test order. For that purpose we work with the HL7 Lab SIG to
                           see how our models could be embedded in their messages. Nevertheless, the new GeneticTestOrder model (presented here for comments
                           only at this point) is an attempt to 'roll-out' the Lab order model and take those classes that we need. In addition it serves
                           as a 'gap analysis' to see if the Lab model can accommodate all the genetic testing requirements. In cases where gaps encountered,
                           there are additions to this model whose origins cannot found in the Lab model. 
                        </p>
                        <p>Therefore, this model serves our dialog with the Lab SIG, as well as a starting point for a generic implementation guide on
                           how to use the Lab model for genetic testing purposes. It could be that this model will be balloted in future ballot cycles,
                           depending on feedback we get from ballotters, early adopters and other stakeholders.   
                        </p>
                        <p><strong>GeneticTestOrder class:</strong></p>
                        <p>
                           The entry point is a GeneticTestOrder class allowing the representation of a genetic test order. It's an observation in RQO
                           (Request) mood code. It is (or may be) fulfilled by zero to many GeneticLoci CMETs (see below for further description on how
                           to represent the result of this order). As aforementioned, this class and its associations are also a "roll-out" of the ObservationRequest
                           class in the A_PlacerOrder CMET of the HL7 Lab Result model.
                           
                        </p>
                        <p><strong>Attributes of the GeneticTestOrder class</strong></p>
                        <ul>
                           <li>id: Instance identifier used to uniquely identify the GeneticTestOrder instance.</li>
                           <li>Code: The GeneticTestOrder.code attribute can hold a code denoting the type of test, for example - an HLA haplotype or a genes
                              panel for a specific disease, etc.
                           </li>
                           <li>text: Free-text description, including a reference mechanism.</li>
                           <li>effectiveTime: Time at which the genetic test was ordered.</li>
                           <li>confidentialityCode: A code that controls the disclosure of information about this genetic test order. Vocabulary- The HL7
                              Vocabulary "Confidentiality" with possible extensions from the Clinical Genomics group (TBD).
                           </li>
                           <li>reasonCode: A code specifying the motivation, cause, or rationale of the ordering this genetic test.</li>
                           <li>statusCode	The status of this genetic test order. Based on the HL7 Vocabulary "ActStatus" with possible extensions from the
                              Clinical Genomics group (TBD).
                           </li>
                           <li>methodCode: The method(s) by which this genetic test should be performed (note - this is optional and even if exists - doesn't
                              necessarily imply the actual method used (which is conveyed through the results in the GeneticLoci and GeneticLocus instances).
                              Vocabulary- the HL7 Vocabulary "ObservationMethod" with possible extensions from the Clinical Genomics group (TBD).
                           </li>
                        </ul>
                        <p><strong>Associations</strong></p>
                        <ul>
                           <li>
                              <p><strong>GeneticLoci CMET:</strong></p>
                              <p>
                                 The GeneticLoci CMET itself can be in request or event mood. If it's in request mood, it serves the representation of requested
                                 observation, e.g., the test order is about requesting to find specific sequence variations in certain genes. If it's in event
                                 mood, then it represents the actual findings/results of the genetic testing.
                                 
                              </p>
                           </li>
                           <li>
                              <p>In general, there are four types of data that can be associated with the test order and they are represented by the following
                                 association classes coming off the GeneticTestOrder class:  
                              </p>
                              <ul>
                                 <li>
                                    <p><strong>component</strong></p>
                                    <p>This is a recursive association back to GeneticTestOrder to allow the representation of '-sub-orders' as components of a complex
                                       order.  
                                    </p>
                                 </li>
                                 <li>
                                    <p><strong>sequelTo</strong></p>
                                    <p>This is a recursive association back to GeneticTestOrder, allowing representation of reflex genetic tests.
                                       
                                    </p>
                                 </li>
                                 <li>
                                    <p><strong>definition</strong></p>
                                    <p>This association connects the order to a master service catalog for elaboration on further details of the ordered test.</p>
                                 </li>
                                 <li>
                                    <p><strong>pertinentInformation</strong></p>
                                    <p>This association connects to the A_SupportingClinicalInformation CMET, which allows the representation of patient specific
                                       data that is important to accompany the test order. This CMET might be replaced by the Clinical Statement model, under development
                                       in HL7.
                                    </p>
                                 </li>
                              </ul>
                           </li>
                        </ul>
                        <p><br /><img class="graphic" src="graphics/POCg_RM000060.gif" alt="Figure 1: Draft of the GeneticTestOrder Model" /><br />Figure 1: Draft of the GeneticTestOrder Model<br /></p>
                        <p></p>
                        <table class="artifactList">
                           <caption>Contained Hierarchical Message Descriptions</caption>
                           <tr class="">
                              <td class="artifactList_desc">Genotype Event</td>
                              <td class="artifactList_code"><a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_HD000050UV01-hmd" class="">POCG_HD000050UV01</a></td>
                           </tr>
                        </table>
                        <p></p>
                     </div>
                  </div>
               </div>
            </div>
            <div class="subSection">
               <div class="header"><span class="icon"> </span><span class="number">2.3</span><span class="title"><a name="hmds-POCG_DO000000UV-Genotype-ic">Hierarchical Message Descriptions</a></span></div>
               <div class="body">
                  <table class="artifactList">
                     <tr>
                        <td class="artifactList_desc">
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e1379" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Hierarchical Message Descriptions (Sorted by Title)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">GeneticLoci(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_HD000050UV01-hmd" class="">POCG_HD000050UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">GeneticLocus(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_HD000010UV01-hmd" class="">POCG_HD000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e13792" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Hierarchical Message Descriptions (Sorted by Structured Sort Name)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Genotype Event(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_HD000010UV01-hmd" class="">POCG_HD000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">Genotype Event(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_HD000050UV01-hmd" class="">POCG_HD000050UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                           <table cellpadding="0" cellspacing="0" style="border:0px" class="help_table">
                              <tr id="d0e13793" class="toc_title_row" onclick="toggle(this);" style="cursor: hand;" onmouseover="highlightRow(this,1);" onmouseout="highlightRow(this,0);">
                                 <td valign="top"><span class="toc_title_arrow">&#9660;</span><span class="toc_title_text"><strong> Hierarchical Message Descriptions (Sorted by Display Order)</strong></span></td>
                              </tr>
                              <tr class="toc_row_master">
                                 <td valign="top">
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">GeneticLocus(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_HD000010UV01-hmd" class="">POCG_HD000010UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                    <table cellpadding="0" cellspacing="0" style="border:0px" class="">
                                       <tr>
                                          <td class="toc_indent_flag_gap"> </td>
                                          <td>
                                             <table style="border:0px" cellspacing="0" cellpadding="0">
                                                <tr>
                                                   <td valign="top" align="left" class="general"><img src="../../support/graphics/pointer.gif" alt="pointer" /></td>
                                                   <td valign="top" align="left" class="general">GeneticLoci(<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_HD000050UV01-hmd" class="">POCG_HD000050UV01</a>)&nbsp;
                                                   </td>
                                                </tr>
                                             </table>
                                          </td>
                                       </tr>
                                    </table>
                                 </td>
                              </tr>
                           </table>
                        </td>
                     </tr>
                  </table>
                  <p></p>
                  <div class="heading"><br /><b class="heading">Reference</b></div>
                  <p>For details on the interpretation of this section, see <a href="../../help/v3guide/v3guide.htm#v3ghmd">the description of HMDs</a> in the Version 3 Guide.
                  </p>
                  <div class="subSubSection">
                     <div class="header"><span class="icon"><img src="../../support/graphics/DSTU.gif" alt="DSTU1" /></span><span class="number">2.3.1</span><span class="title"><a name="POCG_HD000010UV01-hmd">GeneticLocus  (POCG_HD000010UV01) </a></span></div>
                     <div class="body">
                        <p>At this time there is no content for this section.</p>
                        <p></p>
                        <div class="subheading">Base Hierarchical Message Description
                           				<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_RM000010UV01-rmi" class=""><img src="../../support/graphics/rmimview.gif" alt="View R-MIM" height="16" width="16" style="border-width:0" /></a> 
                           				<a href="../../domains/uvcg/editable/POCG_HD000010UV-NoEdit.htm" target="_blank"><img src="../../support/graphics/tableview.gif" alt="Table View" style="border:0px" /></a> <a href="editable/POCG_HD000010UV.xls" target="_blank"><img src="../../support/graphics/excelview.gif" height="16" width="16" alt="Excel View" style="border:0px" /></a> 
                        </div>
                        <table class="hmd">
                           <caption>Message Type List</caption>
                           <tr class="">
                              <td class="hmd_desc"><a name="POCG_MT000010UV01-msg"></a>Genotype Event
                              </td>
                              <td class="hmd_code">POCG_MT000010UV01</td>
                              <td class="hmd_links"><a href="../../domains/uvcg/editable/POCG_HD000010UV-NoEdit.htm" target="_blank"><img src="../../support/graphics/tableview.gif" alt="Table View" style="border:0px" /></a> <a href="../../processable/multicacheschemas/POCG_MT000010UV01.htm" target="_blank"><img src="../../support/graphics/schemaview.gif" height="16" width="16" style="border:0px" alt="Schema View" /></a> 
                                 										 
                                 									
                                 
                              </td>
                           </tr>
                        </table>
                     </div>
                  </div>
                  <div class="subSubSection">
                     <div class="header"><span class="icon"><img src="../../support/graphics/DSTU.gif" alt="DSTU1" /></span><span class="number">2.3.2</span><span class="title"><a name="POCG_HD000050UV01-hmd">GeneticLoci  (POCG_HD000050UV01) </a></span></div>
                     <div class="body">
                        <p>At this time there is no content for this section.</p>
                        <p></p>
                        <div class="subheading">Base Hierarchical Message Description
                           				<a href="../../domains/uvcg/uvcg_Genotype.htm#POCG_RM000050UV01-rmi" class=""><img src="../../support/graphics/rmimview.gif" alt="View R-MIM" height="16" width="16" style="border-width:0" /></a> 
                           				<a href="../../domains/uvcg/editable/POCG_HD000050UV-NoEdit.htm" target="_blank"><img src="../../support/graphics/tableview.gif" alt="Table View" style="border:0px" /></a> <a href="editable/POCG_HD000050UV.xls" target="_blank"><img src="../../support/graphics/excelview.gif" height="16" width="16" alt="Excel View" style="border:0px" /></a> 
                        </div>
                        <table class="hmd">
                           <caption>Message Type List</caption>
                           <tr class="">
                              <td class="hmd_desc"><a name="POCG_MT000050UV01-msg"></a>Genotype Event
                              </td>
                              <td class="hmd_code">POCG_MT000050UV01</td>
                              <td class="hmd_links"><a href="../../domains/uvcg/editable/POCG_HD000050UV-NoEdit.htm" target="_blank"><img src="../../support/graphics/tableview.gif" alt="Table View" style="border:0px" /></a> <a href="../../processable/multicacheschemas/POCG_MT000050UV01.htm" target="_blank"><img src="../../support/graphics/schemaview.gif" height="16" width="16" style="border:0px" alt="Schema View" /></a> 
                                 										 
                                 									
                                 
                              </td>
                           </tr>
                        </table>
                     </div>
                  </div>
               </div>
            </div>
            <p>
               <table>
                  <tbody>
                     <tr>
                        <td><a class="custombutton" href="#top">Return to top of page</a></td>
                     </tr>
                  </tbody>
               </table>
            </p>
         </div>
      </div>
   </body>
</html>